src/parser/shaman/elemental/modules/talents/PrimalFireElemental.js

```diff
@@ -114,7 +109,7 @@ class PrimalFireElemental extends Analyzer {
         icon={<SpellIcon id={SPELLS.FIRE_ELEMENTAL.id} />}
         value={`~ ${formatPercentage(this.damagePercent)} %`}
         label="Of total damage"
-        tooltip={`PFE contributed ${formatNumber(this.damagePerSecond)} DPS (${formatNumber(this.damageGained)} total damage).`}
+        tooltip={`PFE contributed ${formatNumber(this.damagePerSecond)} DPS (${formatNumber(this.damageGained)} total damage) and also generated ${formatNumber(this.maelstromGained)} Maelstrom.`}
       />
     );
   }
```

Comment:
If you want you can also update to use the `position` prop instead of the deprecated `statisticOrder`